packages/attestation-service/src/sms/messagebird.ts

```diff
@@ -61,12 +64,36 @@ export class MessageBirdSmsProvider extends SmsProvider {
     return [bodyParser.urlencoded({ extended: false })]
   }
 
+  async getUSNumbers(): Promise<string[]> {
+    const response = await fetch('https://numbers.messagebird.com/v1/phone-numbers?features=sms', {
+      method: 'GET',
+      headers: {
+        Accept: 'application/json',
+        Authorization: `AccessKey ${this.apiKey}`,
+      },
+    })
+    if (!response.ok) {
+      throw new Error('Could not list numbers! ' + response.status)
+    }
+    const body = JSON.parse(await response.text())
+    return body.items
+      ? body.items
+          .filter((n: any) => n.country === 'US' && n.kycStatus === 'ok')
```

Comment:
oh nice.  interesting that the KYC specific to the number rather than the account.